save-cloud-common/src/jsMain/kotlin/com/saveourtool/save/utils/FileUtils.kt

```diff
@@ -0,0 +1,15 @@
+/**
+ * js implementation for FileUtils - Not implemented
+ */
+
+package com.saveourtool.save.utils
+
+import okio.FileSystem
+import okio.Path
+
+const val NOT_IMPLEMENTED_ON_JS = "Cannot be used in js."
```

Comment:
it cannot be used in legacy JS

Reply:
It cannot be used in js as well as js legacy, can it?

Reply:
yes, you're right. I confused with serialization